src/asciidocParser.ts

```diff
@@ -53,8 +50,18 @@ export class AsciidocParser {
           this.errorCollection.clear();
         }
 
-        const memoryLogger = asciidoctor.MemoryLogger.create();
-        asciidoctor.LoggerManager.setLogger(memoryLogger);
+        const memoryLogger = this.adProcessor.MemoryLogger.create()
+        this.adProcessor.LoggerManager.setLogger(memoryLogger)
+
+        this.registry = this.adProcessor.Extensions.create()
+
+        highlightjsAdapter.register(this.registry)
+
+        const use_kroki = vscode.workspace.getConfiguration('asciidoc', null).get('use_kroki')
```

Comment:
nitpick: use camelCase

```suggestion
        const useKroki = vscode.workspace.getConfiguration('asciidoc', null).get('use_kroki')
```

(it was already in snake_case butit should be camelCase)

Reply:
:+1:

Reply:
Sorry I was only talking about the variable name in the JavaScript code. We probably shouldn't rename the name of the configuration option.
If we do that I guess we will need to "migrate" existing configuration no?

Reply:
Yes, I guess it's unnecessary at this time. The settings are a mix of different casing styles. I will revert.